.github/workflows/PythonLintAndSpell.yml

```diff
@@ -42,18 +42,18 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
       uses: actions/setup-python@v4
       with:
-        python-version: 3.8
+        python-version: 3.9
 
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install git+https://github.com/SimpleITK/SimpleITKSpellChecking@v0.1
+        python -m pip install git+https://github.com/SimpleITK/CommentSpellCheck
```

Comment:
Is there a new tag that we could pin?

Reply:
Yeah, it's currently at release v0.2.0.  I can pin it to that.